index.html

```diff
@@ -227,35 +254,34 @@ <h1 class="about-title">About me</h1>
     <!-- Contact Form Start -->
 
 
-    <footer>
+    <footer id="footer">
 
         <div id="maincontact">
             <div class="contact">
-                <h2>I'm always <br> interested in hearing about new projects, so if you'd like to chat please get in
+                <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in
                     touch.</h2>
             </div>
             <form action="https://formspree.io/f/mrgjwpjk" method="post" class="form-contact">
                 <div id="form-contact">
                     <input type="text" class="name" maxlength="30" required name="name" placeholder="Full name">
-
                     <input type="email" class="email" name="email" required placeholder="Email Address">
                     <textarea class="txt-area" maxlength="500" name="txt-area" required id="txt-area" cols="30" rows="5"
                         placeholder="Enter text here"></textarea>
-                </div>
-                <button class="form" type="submit"> Get in touch</button>
-                <div>
-                    <div id="c-line">
-                        <div></div>
-                    </div>
-                    <div id="s-icons">
-                        <img src="/images/social-media-icon.png" alt="">
-                    </div>
-                    <div class="l-line">
-                        <img src="/images/light.png" alt="">
-                    </div>
+                    <button class="form" type="submit"> Get in touch</button>
                 </div>
             </form>
         </div>
+        <div class="footer">
+            <div id="c-line">
+
+            </div>
+            <div id="s-icons">
+                <img src="/images/social-media-icon.png" alt="">
+            </div>
```

Comment:
- [ ] Your social icons look good as they look on the Figma design but I realized that it is just an image wrapping all the icons. When I tried to click on any of them, It was not possible. Imagine If it was a recruiter who wanted to reach out to you via LinkedIn, you would have lost the opportunity. Please try another way in order to add all those social media icons **which are links to your social media platforms and make sure they are working and they are styled the same way as they look on the Figma design.**
**HINT** 💡 If you look in your `headline` section you will find out that there those social media icons that you need to add in the footer so why not copy and paste their HTML and then change their styles in order to meet what is on the Figma design 😃